.github/workflows/test.yml

```diff
@@ -89,15 +89,10 @@ jobs:
       run: |
         sudo -u testbot bash -lc 'make test'
 
-    - name: Run viam-cartographer cartographer integration tests
-      if: matrix.platform == 'linux/amd64'
-      run: |
-        sudo -u testbot bash -lc 'go test -v -race -run TestCartographerIntegration'
```

Comment:
We no longer need to run full mod tests separately as they no longer depend on a separate executable. `make test` will run integration tests in addition to unit tests.